### ISSUE TITLE
Adding the first page to the basic project results with incorrect app rendered

### ISSUE THREAD

**Issue by adriatic** [redwoodjs/redwood#6777]
### What's not working?

Running the following commands
```
yarn create redwood-app my-redwood-project
cd my-redwood-project
yarn redwood dev
```
results with

<img width="691" alt="image" src="https://user-images.githubusercontent.com/2712405/199609388-b31bcfa2-1aaa-429c-aad2-94e9779c0b42.png">

Now, adding the following command

```
yarn redwood generate page my-page
```

yields

<img width="579" alt="image" src="https://user-images.githubusercontent.com/2712405/199609878-f66e8260-9469-42d0-9391-240fa5771432.png">

The text 
```
You’re seeing this because you don’t have a page at the / path.
Type yarn redwood generate page home / in your CLI to create one.
```
should not be present.

### What's your environment? (If it applies)

```shell
System:
    OS: macOS 13.0
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.15.1 - /private/var/folders/y3/n2p741pj7vq9ckj9jrmlbp6m0000gn/T/xfs-791aea63/node
    Yarn: 3.2.3 - /private/var/folders/y3/n2p741pj7vq9ckj9jrmlbp6m0000gn/T/xfs-791aea63/yarn
  Databases:
    SQLite: 3.39.4 - /usr/bin/sqlite3
  Browsers:
    Chrome: 107.0.5304.87
    Firefox: 105.0.3
    Safari: 16.1
  npmPackages:
    @redwoodjs/core: 3.2.2 => 3.2.2
```

### Are you interested in working on this?

- [X] I'm interested in working on this

**Comment by Josh-Walker-GM** [comment]
> The text
>
> ```
> You’re seeing this because you don’t have a page at the / path.
> Type yarn redwood generate page home / in your CLI to create one.
> ```
> should not be present.

Hey @adriatic why do you believe that this text should not exist?

My understanding was that the `my-page` within the `yarn redwood generate page my-page` command is used to specify the route you wish the new page to be generated at. Since this would not correspond to `/` route then the text seems valid to me?

My understanding of this comes from (among other places) where this `path` option is added to the yargs options: https://github.com/redwoodjs/redwood/blob/4d506cd87984dd6390355b98b8a47ae26d7f9773/packages/cli/src/commands/generate/page/page.js#L149-L154

To me it appears the general new user flow is then:
1. Message that shows there are no pages
2. Add a page at `/my-page`
3. Message that shows your `/my-page` exists but states no page for `/` exists

Unless you run the generator with `/` in place of `/my-page` or you manually register a new page then it seems to be that 3. and the corresponding message is valid?

**Comment by adriatic** [comment]
Hello, @Josh-Walker-GM 

You are correct when citing the explanation rendered after the command
```
yarn redwood generate page my-page
```
resulting with

<img width="400" alt="image" src="https://user-images.githubusercontent.com/2712405/199638299-3b4ac4bb-2c13-4e2b-94ed-d6e69320b8af.png">

is issued -- as at that time, there is still no page at the `root` of the app (page at the / path). However, note that the suggested flow does not create the page at the `root` - only the second time through, the user is advised to run

```
yarn redwood generate page home /
```
which creates the root page and the "Welcome to RedwoodJS" gets removed (finally).

This sequence used to be more straightforward - the Welcome page immediately advised to create the home page at the root. Today I saw the welcome page show up twice and that led me to think that I saw a bug.

All is fine now after your comment made me realize that I jumped the gun.

I purpose to close this issue
